File: CS/GridWithFilterPanelButtons/TestData.cs
using System;
using System.Linq;

namespace GridWithFilterPanelButtons
{
    class TestData
    {
        public TestData(string name, bool check)
        {
            Name = name;
            Check = check;
        }

        public bool Check { get; set; }
        public string Name { get; set; }
    }
}

File: VB/GridWithFilterPanelButtons/TestData.vb
Imports System
Imports System.Linq

Namespace GridWithFilterPanelButtons
    Friend Class TestData
        Public Sub New(ByVal name As String, ByVal check As Boolean)
            Me.Name = name
            Me.Check = check
        End Sub

        Public Property Check() As Boolean
        Public Property Name() As String
    End Class
End Namespace
